draft-tus-httpbis-resumable-uploads-protocol.md

```diff
@@ -312,14 +303,53 @@ If the server does not consider the upload associated with this token active, it
 :authority: example.com
 :path: /upload
 upload-token: :SGVs…SGU=:
-upload-draft-interop-version: 1
+upload-draft-interop-version: 2
 
 :status: 204
 upload-offset: 100
 cache-control: no-store
 ~~~
 
-The client MAY automatically start uploading from the beginning using Upload Transfer Procedure ({{upload-transfer}}) if `404 (Not Found)` status code is received. The client SHOULD NOT automatically retry if a status code other than 204 and 404 is received.
+The client MAY automatically start uploading from the beginning using Upload Creation Procedure ({{upload-creation}}) if `404 (Not Found)` status code is received. The client SHOULD NOT automatically retry if a status code other than 204 and 404 is received.
+
+# Upload Appending Procedure {#upload-appending}
+
+The Upload Appending Procedure is used for resuming an existing upload.
+
+The request MUST use the `HEAD` method and include the `Upload-Token` header. The `Upload-Offset` header field ({{upload-offset}}) MUST be set to the resumption offset.
+
+If the end of the request body is not the end of the upload, the `Upload-Incomplete` header field ({{upload-incomplete}}) MUST be set to true.
+
+The client SHOULD NOT send representation metadata in the request and servers SHOULD ignore any representation metadata received.
```

Comment:
I'm not saying it's wrong but isn't this kind of strange when considering that we mention "Content-Type" as metadata earlier in the draft? Content-Type is most likely automatically added when appending data by the underlying http lib.

Reply:
I removed the client-side requirement